src/audio/codec_adapter/codec/cadence.c

```diff
@@ -383,9 +383,6 @@ int cadence_codec_prepare(struct comp_dev *dev)
 
 	comp_dbg(dev, "cadence_codec_prepare() start");
```

Comment:
We should on our TODO list a task to create documentation for codec_adapter feature including a state machine diagram!!!

Reply:
This should all be aligned with regular processing for v1.9 as per TSC discussion. @ranj063 and @juimonen have started working on this. @cujomalainey fyi.